### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vfdev-5**
Up versions according to https://github.com/pytorch/vision/pull/5106

I cancelled all CI checks as I forgot to put `[skip ci]` in the commit and its MD file only change

cc @NicolasHug

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Instead of relying directly on `black` however, we rely on
 [ufmt](https://github.com/omnilib/ufmt), for compatibility reasons with Facebook
 internal infrastructure.
 
-To format your code, install `ufmt` with `pip install ufmt==1.3.2 black==21.9b0 usort==0.6.4` and use e.g.:
+To format your code, install `ufmt` with `pip install ufmt==1.3.2 black==22.3.0 usort==1.0.2` and use e.g.:
```
Comment (NicolasHug) [inline CONTRIBUTING.md:86]:
```suggestion
To format your code, install `ufmt` with `pip install ufmt==3.9.2 black==22.3.0 usort==1.0.2` and use e.g.:
```

**Review by NicolasHug — APPROVED** [review]
I'll approve in case you prefer to merge this one, but I took care of that in https://github.com/pytorch/vision/pull/6303 (+ the `git blame` ignore thing)

**Comment by vfdev-5** [comment]
I'll close it if you already did that

File: CONTRIBUTING.md (hunk 1 above)
Comment (NicolasHug) [reply CONTRIBUTING.md:86]: My bad I got mixed up with another `hook`, this isn't needed

**Pull request closed without merging**